CHANGELOG.md

```diff
@@ -0,0 +1,128 @@
+# Change Log
+
+## [v0.1](https://github.com/volcano-sh/volcano/tree/v0.1) (2019-05-11)
+**Implemented enhancements:**
+
+- Enable robot for Volcano [\#15](https://github.com/volcano-sh/volcano/issues/15)
```

Comment:
please also highlight other important features.

Reply:
this release notes is auto-generated, it picks up enhancement based on the issue type we create in github issues

Reply:
we're using kind/feature for that, right? 

xref https://github.com/volcano-sh/volcano/issues/15#issuecomment-477409104

Reply:
@k82cn yes it seems like it picks from that, but looks like kind/feature is not supported by https://github.com/github-changelog-generator/github-changelog-generator

Reply:
so what's our next step?

Reply:
we can remove this section, this sections looks like a contradictory one